src/lib.rs

```diff
@@ -1,24 +1,7 @@
-pub use enums::{
-    ChannelType,
-    Copyright,
-    CRC,
-    Emphasis,
-    Error,
-    Genre,
-    Layer,
-    Status,
-    Version,
-};
-pub use metadata::{
-    read_from_file,
-    read_from_slice,
-};
-pub use types::{
-    AudioTag,
-    Frame,
-    MP3Metadata,
-    OptionalAudioTags,
-    Url,
+pub use crate::{
+    enums::{ChannelType, Copyright, Emphasis, Error, Genre, Layer, Status, Version, CRC},
```

Comment:
I strongly dislike this way of writing reexports... Also, what's the point of putting everything inside `pub use crate::`?

Reply:
this was done by "cargo fix --edition" so it's the convention of the Rust language

Reply:
It's definitely not. It's the first time I ever saw it.

Reply:
ok, i just tried it again:

if you run `cargo fix --edition` with cargo 1.47.0 on this:

```rust
mod bar;
mod baz;

pub use bar::*;
pub use baz::*;
```

It transforms to
```rust
mod bar;
mod baz;
 
pub use crate::bar::*;
pub use crate::baz::*;
```

Reply:
That's pretty bad...